qa/L1_videoreader_test/test.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash -e
+
+pushd ../..
+
+cd docs/examples/video
+
+# Install FFmpeg
+apt-get -y install yasm
```

Comment:
Why we need to do it? I think we should ship FFmpeg inside whl. Please update bundle-wheel.sh: DEPS_LIST and DEPS_SONAME. We don't want to force user to install FFmpeg just to use DALI.